core/src/apps/tezos/get_public_key.py

```diff
@@ -1,7 +1,7 @@
 from trezor import ui
 from trezor.messages import ButtonRequestType
 from trezor.messages.TezosPublicKey import TezosPublicKey
-from trezor.ui.text import Text
+from trezor.ui.components.tt.text import Text
```

Comment:
couldn't we use `show_pubkey` here too?

Reply:
Not sure if I overlooked it or didn't use it because of slight difference ("..." at the end if pubkey doesn't fit screen). We definitely should unify those (and make them scrollable if the pubkey doesn't fit), fixed in 48190477e1de3bf7ceca76880c7dcc587819fe42.